images/runtime/php-fpm/template.base.Dockerfile

```diff
@@ -8,15 +8,20 @@ ENV PHP_ORIGIN php-fpm
 ENV NGINX_RUN_USER www-data
 # Edit the default DocumentRoot setting
 ENV NGINX_DOCUMENT_ROOT /home/site/wwwroot
-# Install NGINX 
-RUN apt-get update \
-    && apt-get install nano nginx -y
+# Install NGINX latest stable version using APT Method with Nginx Repository instead of distribution-provided one:
+# - https://www.linuxcapable.com/how-to-install-latest-nginx-mainline-or-stable-on-debian-11/
+RUN apt-get update
+RUN apt install curl nano -y
+RUN curl -sSL https://packages.sury.org/nginx/README.txt | bash -x
+RUN apt-get update
+RUN yes '' | apt-get install nginx-core nginx-common nginx nginx-full -y
```

Comment:
I saw that apt-get previously had a `--force-yes` option, that is now deprecated but seems to be replaced by even more precise `--allow-unauthenticated`, `--allow-downgrades`, `--allow-remove-essential`, `--allow-change-held-packages` parameters. Would it be possible to use these to be more granular than a blanket `yes` command?

Reply:
This was a tricky place that I was working. There
are two prompts that showing options and let users enter either `"N"(default)` or some other letters. This is different from other `Y/N `prompts that we've seen. I tried lots of ways and the `force-yes` doesn't seem to work. And I found that `yes ' '` will perform similar as we tab the enter key and it will work.

Reply:
OK, thanks for clarifying!